datadog_checks_base/datadog_checks/base/stubs/aggregator.py

```diff
@@ -37,6 +37,22 @@ def backend_normalize_metric_name(metric_name):
     return METRIC_DOTUNDERSCORE_CLEANUP.sub(".", metric_name).strip("_")
 
 
+def check_tag_names(metric, tags):
+    forbidden_tags = [
+        'cluster_name',
+        'clustername',
+        'cluster',
+        'host_name',
+        'hostname',
+        'host',
```

Comment:
`host` related tags should also be more specific like `apache_host`, or should be used as hostname attribute instead

Reply:
These too? https://docs.datadoghq.com/getting_started/tagging/

Reply:
A lot of integrations are failing because of these, we should add them in separate prs

Reply:
maybe we should also add the [Unified Service Tagging
](https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging/?tab=kubernetes) : `service`,`version`,`env`

Reply:
A lot of openmetrics integrations are sending these tags https://dev.azure.com/datadoghq/integrations-core/_build/results?buildId=68132&view=results, so I prefer to separate it